main.go

```diff
@@ -302,6 +303,48 @@ func tr1d1um(arguments []string) (exitCode int) {
 	return 0
 }
 
+func loadTracing(v *viper.Viper, appName string) (candlelight.Tracing, error) {
+	var tracing = candlelight.Tracing{
+		Enabled:        false,
+		Propagator:     propagation.TraceContext{},
+		TracerProvider: trace.NewNoopTracerProvider(),
+	}
+	if v.IsSet(tracingConfigKey) {
```

Comment:
Can't you just `Unmarshal` the `candlelight.Tracing` key instead of checking first?